### ISSUE TITLE
Issue with UPPER_UNKNOWN_NNP_UPPERCASE_SEQ

### ISSUE THREAD

**Issue by KnowZero** [languagetool-org/languagetool#752]
Examples:

I stayed at the Holday Inn Express. = no spelling mistake
I stayed at the holday Inn Express. = spelling mistake
I stayed at the holday inn express. = spelling mistake
I went on a Holday again. = spelling mistake

It seems it ignores spelling mistakes when used in names of locations when there are multiple capital letters in sentences.

**Comment by danielnaber** [comment]
It's the combination of `UPPER_UNKNOWN_NNP_UPPERCASE_SEQ` and `NNP_MID_END_SENTENCE` - the latter one causes the word to be ignored by the spell checker.

**Comment by KnowZero** [comment]
I see, so will this require a rule change or give an option to disable it if someone wants?

**Comment by danielnaber** [comment]
@MikeUnwalla What do you think?

**Comment by MikeUnwalla** [comment]
If a word is a proper noun, then by default, the spell checker should not give a warning. Both 'Holiday Inn Express' and 'Holday Inn Express' are proper nouns.

If we remove rulegroup IGNORE_SPELLING_OF_NNP, we will get many false warnings. Refer to https://github.com/languagetool-org/languagetool/issues/568 and to the comments at the top of the rulegroup.

But, @KnowZero wants to find a misspelled word that is part of a proper noun (http://forum.languagetool.org/t/tinymce-4-plugin/1913/56). To find this type of problem is useful.

I tried to make a grammar rule that finds NNP (proper nouns) that are not standard English. This is the pattern (http://wiki.languagetool.org/tips-and-tricks#toc25):
`<token postag="NNP"><exception negate_pos="yes" postag="NNP"/></token>`

The rule does not work, because the disambiguator removes postags from some proper nouns. For the example sentence, "I stayed at the Holiday Inn Express", the rule gives warnings for 'Holiday' and 'Inn', because the disambiguator removes these postags:
UPPER_NNP removes NN:UN from 'Holiday'.
UPPER_NN_NNP removes NN from 'Inn'.

An option to disable a disambiguator rule could help. Although I vaguely remember a discussion some years ago about the problems that that could cause, I think that we could safely disable the rule IGNORE_SPELLING_OF_NNP.

Possibly, a quick fix for @KnowZero is to remove IGNORE_SPELLING_OF_NNP from the disambiguator.
